package.json

```diff
@@ -19,9 +19,14 @@
     "@ag-grid-community/infinite-row-model": "^32.0.2",
     "@ag-grid-community/react": "^32.0.2",
     "@ag-grid-community/styles": "^32.0.2",
+    "@auth0/auth0-react": "^2.2.4",
+    "@headlessui/react": "^2.1.8",
     "@remix-run/cloudflare": "^2.9.2",
     "@remix-run/cloudflare-pages": "^2.9.2",
-    "@remix-run/react": "^2.9.2",
+    "@remix-run/react": "^2.11.2",
+    "@tailwindcss/forms": "^0.5.9",
+    "ag-grid-community": "^32.1.0",
```

Comment:
AG Grid has [two ways of being imported](https://www.ag-grid.com/javascript-data-grid/modules/), a packages approach and a modules approach- we're already using the modules approach, and this adds an import of the packages approach too. We should be able to drop this and refer to the appropriate `@ag-grid-community/{name}` package wherever we're using this.